api_app/api/routes/operations.py

```diff
@@ -0,0 +1,17 @@
+
+from fastapi import APIRouter, Depends
+
+from db.repositories.operations import OperationRepository
+from api.dependencies.database import get_repository
+from models.schemas.operation import OperationInList
+from resources import strings
+from services.authentication import get_current_tre_user_or_tre_admin
+
+
+operations_router = APIRouter(dependencies=[Depends(get_current_tre_user_or_tre_admin)])
+
+
+@operations_router.get("/operations/my", response_model=OperationInList, name=strings.API_GET_MY_OPERATIONS)
```

Comment:
Is there a reason we need the "my" path?
Could the route `/operations/` not do the job?

Reply:
yea possibly - this was a bit of speculative future proofing really, but maybe just `/operations` would be clearer, I can change it

Reply:
It's fine as is. This comment is linked to the hard-coding of "my" in the UI, and I think that's the more important thing to fix.

Reply:
i've taken out the `my` - Stuart didn't like it either :)